sdk/storage/azure-storage-blob/README.md

```diff
@@ -331,6 +331,8 @@ Defaults to `False`.
 Use the following keyword arguments when instantiating a client to configure encryption:
```

Comment:
Do we want to add a message here that we recommend _not_ using client-side encryption, and link to docs or a guide to migrate to server side encryption? I would also probably put a more detailed note here about the deprecation, you could even reuse the note in the changelog.

Reply:
Yeah, that might be a good idea. We are still internally discussing all messaging around this feature, and I will likely be changing CHANGELOG/warnings/etc. so I'll make another pass at this section.